collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Description: alarms netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from json import loads
+
+from bases.FrameworkServices.UrlService import UrlService
+
+priority = 200200
+update_every = 5
```

Comment:
```suggestion
update_every = 10
```

default alarms `every` is 10